From ae6b78f89155cca68d83160a993ea60dcabe0144 Mon Sep 17 00:00:00 2001
From: Masami Ichikawa <masami.ichikawa@miraclelinux.com>
Date: Thu, 30 May 2019 17:08:24 +0900
Subject: [PATCH] Do not use MULTIARCH value in pkgconfig directory path

Other packages doesn't use such path, the use ${libdir}/pkgconfig.

Signed-off-by: Masami Ichikawa <masami.ichikawa@miraclelinux.com>
---
 Makefile.pre.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.pre.in b/Makefile.pre.in
index c5fc565..3e05cd4 100644
--- a/Makefile.pre.in
+++ b/Makefile.pre.in
@@ -1480,7 +1480,7 @@ inclinstall:
 LIBPL=		@LIBPL@
 
 # pkgconfig directory
-LIBPC=		$(LIBDIR)/$(MULTIARCH)/pkgconfig
+LIBPC=		$(LIBDIR)/pkgconfig
 
 libainstall:	@DEF_MAKE_RULE@ python-config
 	@for i in $(LIBDIR) $(LIBPL) $(LIBPC); \
-- 
2.20.1

